client/android/app/build.gradle

```diff
@@ -84,7 +84,7 @@ def enableProguardInReleaseBuilds = false
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    buildToolsVersion '25.0.0'
```

Comment:
Where do you see that? I've been building / running from Android Studio and haven't seen it screaming about anything...